src/test/java/com/google/step/servlets/PromoCodeServletTest.java

```diff
@@ -0,0 +1,12 @@
+package com.google.step.servlets;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+// @RunWith(JUnit4.class)
+public final class PromoCodeServletTest {
+
```

Comment:
nit: add comment todo saying implementation is coming.

Reply:
Added.